modules/services/sxhkd.nix

```diff
@@ -57,44 +57,18 @@ in
           i3-msg {workspace,move container to workspace} {1-10}
       '';
     };
-
-    extraPath = mkOption {
-      default = "";
-      type = types.envVar;
-      description = ''
-        Additional <envar>PATH</envar> entries to search for commands.
-      '';
-      example = "/home/some-user/bin:/extra/path/bin";
-    };
   };
 
   config = mkIf cfg.enable {
-    home.packages = [ pkgs.sxhkd ];
+    home.packages = [ cfg.package ];
 
     xdg.configFile."sxhkd/sxhkdrc".text = concatStringsSep "\n" [
       keybindingsStr
       cfg.extraConfig
     ];
 
-    systemd.user.services.sxhkd = {
-      Unit = {
-        Description = "simple X hotkey daemon";
-        After = [ "graphical-session-pre.target" ];
-        PartOf = [ "graphical-session.target" ];
-      };
-
-      Service = {
-        Environment =
-          "PATH="
-          + "${config.home.profileDirectory}/bin"
-          + optionalString (cfg.extraPath != "") ":"
-          + cfg.extraPath;
-        ExecStart = "${cfg.package}/bin/sxhkd ${toString cfg.extraOptions}";
-      };
-
-      Install = {
-        WantedBy = [ "graphical-session.target" ];
-      };
-    };
+    xsession.initExtra = ''
+    ${cfg.package}/bin/sxhkd ${toString cfg.extraOptions} &
```

Comment:
Also, perhaps it would be possible to use `systemd-cat` to at least capture any output?

Reply:
Another alternative, I wonder if it would be possible to use something like
```shell
systemd-cat -t sxhkd systemd-run --user --scope -u sxhkd \
  ${cfg.package}/bin/sxhkd ${toString cfg.extraOptions} &
```

Reply:
It is possible, I can see output in the journal and have some .scope unit info.